Add decodeMultiple function to JsonDecoder

fetzi opened the pull request:
To be able to transform an array of objects the decodeMultiple function
is introduced.

fixes #14

Commit:
Add decodeMultiple function to JsonDecoder

To be able to transform an array of objects the decodeMultiple function
is introduced.

fixes #14

Commit:
Apply linter fixes

Commit:
Update readme with decodeMultiple

add documentation about decodeMultiple functionality

jaylinski commented on readme.md:
````diff
@@ -53,12 +53,12 @@ The transformer interface defines two methods:
 ```php
 class PersonTransformer implements Transformer {
-    
+
````
Wieso nicht gleich PSR-2 formatiert?

jaylinski commented on readme.md:
```diff
@@ -81,6 +81,17 @@ The `JsonDecoder` class accepts two boolean constructor parameters to enable the
 To do so a so called `PropertyAccessor` will be installed and on property set the proxy will set the property to accessible, set the according value and then will set the property to not accessible again.
 
+### Transforming an array of elements
+If your json contains an array of elements at the root level you can use the `decodeMultiple` method to transform the json data into an array of class type objects.
```
Ich würd JSON überall groß schreiben.

jaylinski commented on src/JsonDecoder.php:
```diff
@@ -40,6 +40,18 @@ public function decode($jsonString, $classType)
         return $this->decodeArray(json_decode($jsonString, true), $classType);
     }
 
+    public function decodeMultiple($jsonString, $classType)
```
Könntest auch so schreiben: `public function decodeMultiple(string $json, string $class)`

Commit:
Apply fixes from PR

exchange lower case json with JSON in readme. add type hints for decode
and decodeMultiple
